src/App.test.js

```diff
@@ -168,6 +168,7 @@ describe('App component', () => {
               lang: 'es',
               avatar: {},
               plan: {},
+              sms: {},
```

Comment:
In all scenarios are you sure back end always returns an sms object?

Reply:
I guess so. Tests fail if I don't add sms object.

Reply:
But we always get sms object from BE? otherwise you should add a test to allow not having it. And also check if the app breaks.

Reply:
I didn't see any case without this info. Even in free accounts we receive this value empty. I can apply a test to verify what you say.

Reply:
Great, then no need. Thanks! ❤️